scio-extra/src/main/scala/com/spotify/scio/extra/voyager/syntax/SCollectionSyntax.scala

```diff
@@ -100,8 +100,12 @@ class VoyagerPairSCollectionOps(
     require(!uri.exists, s"Voyager URI ${uri.value} already exists")
 
     self.transform { in =>
-      in.reifyAsIterableInGlobalWindow
-        .map { xs =>
+      // Do not use reifyAsIterableInGlobalWindow
+      // as iterableCoder may materialize the full collection
+      self.context
+        .parallelize(Seq((): Unit))
```

Comment:
out of curiosity, how does Dataflow render this job graph? I guess it adds another root-level transform for `.parallelize()` that feeds into the Voyager SCollection?

Reply:
yes: 
![image](https://github.com/spotify/scio/assets/2845540/df6e2da1-d7be-40a1-ba59-7e8fa1276a7c)
I'm just getting rid on the extra `map` from `reifyAsIterableInGlobalWindow` that can trigger the `iterableCoder` forcing the side input collection to fit in memory. Here we directly access the view that should be able to stream side input records from the view